.circleci/config.yml

```diff
@@ -9,8 +9,9 @@ workflows:
     jobs:
       - test-ubuntu18
```

Comment:
As ubuntu18 is also tested in Github Actions, how about disabling this too?

Reply:
I think this is for use_conda=true so I keep it as is.